index.html

```diff
@@ -72,8 +70,105 @@ <h3 class="ability_header">Skills</h3>
                 <li><i class="fa-brands fa-angellist"></i></li>
             </ul>
         </section>
-        <section>
+        <section class="work">  
+            <h2>My recent work</h2>
             
-        </section>
-</body>
-</html>
\ No newline at end of file
+            <div class="work_one">
+                <img src="placeholder.png" alt="image placeholder">
+                <div class="supporting_text">
+                    <h3>Multi-Post Stories</h3>
+                    <p class="work_info">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                    <ul class="languages">
+                        <li>css</li>
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button type="button" class="btn">See Project</button>
+                </div>
+            </div>
```

Comment:
Kindly make sure your card looks like the Figma design. Add margins (top, bottom, left, right) defined in the design as defined in the design. You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma]
## your design
![image](https://user-images.githubusercontent.com/95297251/185715371-5415b0e4-8a0a-4b98-952a-2a9badb7be2a.png)



## Figma design

![image](https://user-images.githubusercontent.com/95297251/185518100-40a57c25-e2ad-4de4-a808-44c615dd5872.png)